homeassistant/components/automation/__init__.py

```diff
@@ -314,6 +314,9 @@ async def reload_service_handler(service_call: ServiceCall) -> None:
 class BaseAutomationEntity(ToggleEntity, ABC):
     """Base class for automation entities."""
 
+    _recorder_excluded_attributes = frozenset(
+        (ATTR_LAST_TRIGGERED, ATTR_MODE, ATTR_CUR, ATTR_MAX, CONF_ID)
+    )
```

Comment:
I'm not sure entities should know about the recorder.

Maybe just `_excluded_attributes` or `_transient_attributes` or `_ephemeral_attributes` or `_run_time_attributes` ?